backend/src/web-site/models/News.ts

```
@@ -0,0 +1,44 @@
+import mongoose, { Document, Schema } from 'mongoose';
+import { ICategory } from './Category';
+
+export interface INews extends Document {
+  _id: String | null;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)